presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -80,6 +80,7 @@
 import com.google.common.collect.Maps;
```

Comment:
Can you please a test for this? You need to create in hive that is unsupported in Presto, this gives you two options. Either use hive integration tests (see ./presto-hive/src/test/sql/create-test.sql and presto-hive-hadoop2/bin/run_hive_tests.sh) or product tests (something like presto-product-tests/src/main/resources/sql-tests/datasets/table_with_header.ddl and presto-product-tests/src/main/resources/sql-tests/testcases/tables_with_header_and_footer.sql)